cle/loader.py

```diff
@@ -365,8 +365,10 @@ def _check_object_memory(obj_):
         if not obj.min_addr <= addr <= obj.max_addr:
             return None
         if not membership_check:
+            self._last_object = obj
             return obj
         if not obj.has_memory:
+            self._last_object = obj
```

Comment:
what's this here for?

Reply:
For example, `NamedRegion`. We could theoretically hit this case when working on embedded binaries.